app/grandchallenge/components/models.py

```diff
@@ -1962,6 +1962,30 @@ def update_size_in_storage(self):
             self.size_in_registry = self.calculate_size_in_registry()
 
 
+class CIVSetContainerMixin:
```

Comment:
Why "container" in the name here? I find that a bit confusing. Not the catchiest, but `CIVSetStringRepresentationMixin` describes clearly what it does. What do you think?

Reply:
Works.

Reply:
I thought it could function as a general spot to put stuff in for CIVSets but I guess that should likely not be a Mixin =/